<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationForm</class>
 <widget class="QWidget" name="TransformationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="imageHLayout">
     <item>
      <widget class="QLabel" name="imageLabel">
       <property name="text">
        <string>Image   </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="imageCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="flipGroup">
     <property name="title">
      <string>Flip</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="flipXButton">
        <property name="text">
         <string>Flip X</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="flipYButton">
        <property name="text">
         <string>Flip Y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="flipZButton">
        <property name="text">
         <string>Flip Z</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="mainSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
